src/components/TodoContainer.js

```diff
@@ -1,114 +1,118 @@
-import React from "react"
-import TodosList from "./TodosList";
-import Header from "./Header"
-import InputTodo from "./InputTodo"
-import { Route, Switch } from "react-router-dom"
- import About from "../pages/About"
-import NotMatch from "../pages/NotMatch"
-import Navbar from "./Navbar"
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/state-in-constructor */
+/* eslint-disable react/no-access-state-in-setstate */
+/* eslint-disable react/sort-comp */
+/* eslint-disable react/no-unused-state */
+/* eslint-disable no-param-reassign */
```

Comment:
Linters rules purpose is to force you to code using good ways. You should disable rules only for really special cases. Else, you should try to stick on linters warning or errors.